### PR TITLE
ENH make _cals computation the same across modules

### PR DESCRIPTION
**Pull request by teonlamont**
Make `_cals` computation consistent across modules. comment brought up in #1952.

### CODE REVIEW

File: mne/io/kit/kit.py [hunk 1]
```diff
@@ -90,10 +90,9 @@ def __init__(self, input_fname, mrk=None, elp=None, hsp=None, stim='>',
         # meg channels plus synthetic channel
         info['nchan'] = self._sqd_params['nchan'] + 1
 
-        cals = np.ones(info['nchan'])
         last_samps = [self._sqd_params['nsamples'] - 1]
         super(RawKIT, self).__init__(
-            info, cals=cals, last_samps=last_samps, verbose=verbose)
+            info, cals=None, last_samps=last_samps, verbose=verbose)
```
Comment (Eric89GXL) [inline mne/io/kit/kit.py]: If you are unifying the computation, the `_BaseRaw` init probably doesn't even need a `cals` parameter anymore.

**Comment by teonlamont** [comment]
good point

**Comment by Eric89GXL** [comment]
Assuming Travis is happy, you done? Looks complete to me.

**Comment by teonlamont** [comment]
i'm done here.

**Comment by Eric89GXL** [comment]
you have some legitimate failures

https://travis-ci.org/mne-tools/mne-python/jobs/59049523#L698

**Comment by teonlamont** [comment]
fixed

**Comment by agramfort** [comment]
thanks @teonlamont

**Pull request merged**